composer.json

```diff
@@ -10,20 +10,22 @@
     ],
     "require": {
         "jms/metadata": "~1.5",
-        "symfony/property-access": "~2.4",
+        "symfony/property-access": "~2.6||~3.0",
```

Comment:
symfony 2.4 is death (: i think it would be no problem should i revert this to 2.4?